src/UI/GameOptionBool.java

```diff
@@ -0,0 +1,17 @@
+package UI;
+
+
+public class GameOptionBool extends GameOption<Boolean>
+{
+  public GameOptionBool(String name, boolean defaultValue)
+  {
+    super(name, new Boolean[] { false, true }, (defaultValue ? 1 : 0));
+  }
+
+  @Override
+  public String getCurrentValueText()
+  {
+    boolean value = optionList.get(getSelectionNormalized()-minOption);
```

Comment:
The `minOption` feels unnecessary here, seeing as you control what that is (to always be zero)

Reply:
True. Removed.